src/server/rest/v1/service/security/SettingSecurity.ts

```diff
@@ -186,6 +186,23 @@ export default class SettingSecurity {
             settings.content.asset.connections.push(sanitizedConnection);
           }
           break;
+        case CryptoSettingsType.CRYPTO:
+          settings.content.crypto = {
+            key: sanitize(request.content.crypto.key),
+            keyProperties: {
+              blockCypher: sanitize(request.content.crypto.keyProperties?.blockCypher),
+              blockSize: sanitize(request.content.crypto.keyProperties?.blockSize),
+              operationMode: sanitize(request.content.crypto.keyProperties?.operationMode)
+            },
+            formerKey: sanitize(request.content.crypto.formerKey),
+            formerKeyProperties: {
+              blockCypher: sanitize(request.content.crypto.formerKeyProperties?.blockCypher),
+              blockSize: sanitize(request.content.crypto.formerKeyProperties?.blockSize),
+              operationMode: sanitize(request.content.crypto.formerKeyProperties?.operationMode)
+            },
+            migrationToBeDone: sanitize(request.content.crypto.migrationToBeDone)
```

Comment:
That value shall not be set by a REST endpoint call, the server shall be the only responsible that set it by detecting if a key change have occurred or not in the call. 

Since it seems the key are hashed in the request payload, you can compare the hashed value of the key and the formerKey to know if the have changed or not.